File: week 6/sentimental-cash/cash.py
from cs50 import get_float
from math import floor

while True:
    change = get_float("Change: ") * 100
    if change > 0:
        break

coins = 0

coins += floor(change / 25)
change -= floor(change / 25) * 25

coins += floor(change / 10)
change -= floor(change / 10) * 10

coins += floor(change / 5)
change -= floor(change / 5) * 5

coins += floor(change / 1)
change -= floor(change / 1) * 1

print(coins)

File: week 1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void){
    int money;
    do {
        money = get_int("Change owed: ");
    }
    while (money < 0);

    int coins = 0;

    while(money>=25){
        money = money - 25;
        coins++;
    }
    while(money>=10){
        money = money - 10;
        coins++;
    }
    while(money>=5){
        money = money - 5;
        coins++;
    }
    while(money>=1){
        money = money - 1;
        coins++;
    }

    printf("%i\n", coins);
}
